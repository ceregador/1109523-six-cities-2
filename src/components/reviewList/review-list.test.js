import React from 'react';
import Review from '../review/review.jsx';
import ReviewList from './review-list.jsx';
import Renderer from 'react-test-renderer';

jest.mock(`../review/review.jsx`);

it(`renders correctly`, () => {
  const tree = Renderer
      .create(
          <ReviewList
            reviews={[{
              rating: 1,
              user: {
                name: ``,
                avatarUrl: ``
              },
              comment: ``,
              reviewData: ``
            }]}
          />)
      .toJSON();

  expect(Review).toHaveBeenCalled();
  expect(tree).toMatchSnapshot();
});
