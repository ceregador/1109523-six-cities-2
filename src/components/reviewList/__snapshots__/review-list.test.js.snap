// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<section
  className="property__reviews reviews"
>
  <h2
    className="reviews__title"
  >
    Reviews Â· 
    <span
      className="reviews__amount"
    >
      1
    </span>
  </h2>
  <ul
    className="reviews__list"
  />
</section>
`;
